### ISSUE TITLE
1.6 - Random Reboots

### ISSUE THREAD

**Issue by strowi** [rancher/rancher#17577]
**What kind of request is this (question/bug/enhancement/feature request):**
BUG

**Steps to reproduce (least amount of steps as possible):**
Setup Rancher + K8, put some workload on it (in our case a nodejs image-processor, scaled to ~16 pods)

**Result:**
Single Nodes randomly rebooting

**Other details that may be helpful:**
please see https://github.com/rancher/os/issues/2538 , since we initially thought the OS might be the problem.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): 1.6.25
- Installation option (single install/HA): Single Install

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Custom
- Machine type (cloud/VM/metal) and specifications (CPU/memory): Bare-Metal >=24CPU / >=128GB RAM, 3*Proxmox VM for etcd + operator.

- Kubernetes version (use `kubectl version`):
```
Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.5-rancher1", GitCommit:"44636ddf318af0483af806e255d0be4bb6a2e3d4", GitTreeState:"clean", BuildDate:"2018-12-04T04:28:34Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker (use `docker info`):

```
Containers: 149
 Running: 98
 Paused: 0
 Stopped: 51
Images: 181
Server Version: 17.03.2-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 4ab9917febca54791c5f071a9d1f404867857fcc
runc version: 54296cf40ad8143b62dbcaa1d90e520a2136ddfe
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-39-generic
Operating System: Ubuntu 18.04.1 LTS
OSType: linux
Architecture: x86_64
CPUs: 40
Total Memory: 125.6 GiB
Name: ranch04
ID: STI5:2YPR:JZW7:AXIU:J4TU:XGCJ:BV5R:2TJ6:IWSA:34JS:PT3W:I4L6
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```

**Comment by strowi** [comment]
After a reboot i can see lots of pod events with 
```
Failed create pod sandbox: rpc error: code = Unknown desc = [failed to set up sandbox container  network for pod : NetworkPlugin cni failed to set up pod  network: No MAC address found, failed to clean up sandbox container "a043a2db53f5a17bf5a88af4b794b87f2d8eb20961770905d8ca1e9e91047372" network for pod "server-k8-6df8cc988c-64kc8": NetworkPlugin cni failed to teardown pod  network: failed to get IP addresses for "eth0": <nil>]
```

**Comment by strowi** [comment]
still happening with rancher-1.6.26 and k8s-1.12.5 ...

**Comment by strowi** [comment]
Ok, finally got a screenshot of the kernel:
<img width="1155" alt="screen" src="https://user-images.githubusercontent.com/589662/53194392-88dd6500-3613-11e9-8093-1116b2871c3e.png">

This looks similar to https://github.com/rancher/os/issues/2428 doesn't it?
```
~> etcd05:~$ uname -a
Linux etcd05 4.15.0-39-generic #42-Ubuntu SMP Tue Oct 23 15:48:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by superseb** [comment]
Looks like https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1801878 and should be fixed in kernel `4.15.0-42.45`.

Let me know if that doesn't solve the issue.
